src/cmd_line/commands/set.ts

```diff
@@ -179,13 +180,15 @@ export class SetCommand extends ExCommand {
           if (type === 'boolean') {
             configuration[option] = true;
           } else {
-            this.showOption(vimState, option, currentValue);
+            if(vimState)
```

Comment:
Looks like `prettier` didn't run

Reply:
Done. Though prettier changed more lines now for some reason.

Reply:
This should be handled by our CI but it's been broken for a while now